diff --git a/CMakeLists.txt b/CMakeLists.txt
index 0a1f95830e..528eede3fe 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1445,6 +1445,7 @@ if(ENABLE_PLUGINS)
 		plugins/epan/mate
 		plugins/epan/opcua
 		plugins/epan/profinet
+		plugins/epan/sap
 		plugins/epan/stats_tree
 		plugins/epan/transum
 		plugins/epan/unistim
diff --git a/packaging/nsis/wireshark.nsi b/packaging/nsis/wireshark.nsi
index 710042b391..ad6851e35c 100644
--- a/packaging/nsis/wireshark.nsi
+++ b/packaging/nsis/wireshark.nsi
@@ -989,6 +989,7 @@ File "${STAGING_DIR}\plugins\${VERSION_MAJOR}.${VERSION_MINOR}\epan\gryphon.dll"
 File "${STAGING_DIR}\plugins\${VERSION_MAJOR}.${VERSION_MINOR}\epan\irda.dll"
 File "${STAGING_DIR}\plugins\${VERSION_MAJOR}.${VERSION_MINOR}\epan\opcua.dll"
 File "${STAGING_DIR}\plugins\${VERSION_MAJOR}.${VERSION_MINOR}\epan\profinet.dll"
+File "${STAGING_DIR}\plugins\${VERSION_MAJOR}.${VERSION_MINOR}\epan\sap.dll"
 File "${STAGING_DIR}\plugins\${VERSION_MAJOR}.${VERSION_MINOR}\epan\unistim.dll"
 File "${STAGING_DIR}\plugins\${VERSION_MAJOR}.${VERSION_MINOR}\epan\wimax.dll"
 File "${STAGING_DIR}\plugins\${VERSION_MAJOR}.${VERSION_MINOR}\epan\wimaxasncp.dll"
