commit 897e0017d6dfba2a8045ba858e018e161c108d49
Author: Martin Gallo <mgallo@coresecurity.com>
Date:   Fri Jan 23 17:09:43 2015 -0300

    Configured sap plugin.

diff --git a/CMakeLists.txt b/CMakeLists.txt
index eb50acc..ee904b4 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -704,6 +704,7 @@ if(ENABLE_PLUGINS)
 		plugins/mate
 		plugins/opcua
 		plugins/profinet
+		plugins/sap
 		plugins/stats_tree
 		plugins/unistim
 		plugins/wimax
diff --git a/Makefile.am b/Makefile.am
index d813784..1557e65 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -306,6 +306,7 @@ plugin_ldadd = $(_CUSTOM_plugin_ldadd_) \
 	-dlopen plugins/mate/mate.la \
 	-dlopen plugins/opcua/opcua.la \
 	-dlopen plugins/profinet/profinet.la \
+	-dlopen plugins/sap/sap.la \
 	-dlopen plugins/stats_tree/stats_tree.la \
 	-dlopen plugins/unistim/unistim.la \
 	-dlopen plugins/wimax/wimax.la \
diff --git a/configure.ac b/configure.ac
index e6aa52f..e175a7b 100644
--- a/configure.ac
+++ b/configure.ac
@@ -2966,6 +2966,7 @@ AC_OUTPUT(
   plugins/mate/Makefile
   plugins/opcua/Makefile
   plugins/profinet/Makefile
+  plugins/sap/Makefile
   plugins/stats_tree/Makefile
   plugins/unistim/Makefile
   plugins/wimax/Makefile
diff --git a/epan/Makefile.am b/epan/Makefile.am
index 0739c6d..5bc372f 100644
--- a/epan/Makefile.am
+++ b/epan/Makefile.am
@@ -235,6 +235,13 @@ plugin_src = \
 	../plugins/irda/packet-irda.c \
 	../plugins/m2m/packet-m2m.c \
 	../plugins/m2m/wimax_tlv.c \
+	../plugins/sap/src/packet-sapdiag.c \
+	../plugins/sap/src/packet-sapenqueue.c \
+	../plugins/sap/src/packet-saprouter.c \
+	../plugins/sap/src/packet-sapprotocol.c \
+	../plugins/sap/src/packet-saprfc.c \
+	../plugins/sap/src/packet-sapms.c \
+	../plugins/sap/src/packet-sapsnc.c \
 	../plugins/wimax/crc.c \
 	../plugins/wimax/crc_data.c \
 	../plugins/wimax/mac_hd_generic_decoder.c \
diff --git a/packaging/nsis/Makefile.nmake b/packaging/nsis/Makefile.nmake
index eb037e6..6040b32 100644
--- a/packaging/nsis/Makefile.nmake
+++ b/packaging/nsis/Makefile.nmake
@@ -51,6 +51,7 @@ PLUGINS= \
 	../../plugins/mate/mate.dll \
 	../../plugins/opcua/opcua.dll \
 	../../plugins/profinet/profinet.dll \
+	../../plugins/sap/sap.dll \
 	../../plugins/stats_tree/stats_tree.dll \
 	../../plugins/unistim/unistim.dll \
 	../../plugins/wimax/wimax.dll \
diff --git a/packaging/nsis/wireshark.nsi b/packaging/nsis/wireshark.nsi
index 6d4455b..6a62483 100644
--- a/packaging/nsis/wireshark.nsi
+++ b/packaging/nsis/wireshark.nsi
@@ -983,6 +983,7 @@ File "${STAGING_DIR}\plugins\${VERSION}\irda.dll"
 File "${STAGING_DIR}\plugins\${VERSION}\m2m.dll"
 File "${STAGING_DIR}\plugins\${VERSION}\opcua.dll"
 File "${STAGING_DIR}\plugins\${VERSION}\profinet.dll"
+File "${STAGING_DIR}\plugins\${VERSION}\sap.dll"
 File "${STAGING_DIR}\plugins\${VERSION}\unistim.dll"
 File "${STAGING_DIR}\plugins\${VERSION}\wimax.dll"
 File "${STAGING_DIR}\plugins\${VERSION}\wimaxasncp.dll"
diff --git a/plugins/Makefile.am b/plugins/Makefile.am
index 0cca3cd..a588147 100644
--- a/plugins/Makefile.am
+++ b/plugins/Makefile.am
@@ -30,6 +30,7 @@ SUBDIRS = $(_CUSTOM_SUBDIRS_) \
 	mate \
 	opcua \
 	profinet \
+	sap \
 	stats_tree \
 	unistim \
 	wimax \
diff --git a/plugins/Makefile.nmake b/plugins/Makefile.nmake
index 2bf5a9c..6e675a7 100644
--- a/plugins/Makefile.nmake
+++ b/plugins/Makefile.nmake
@@ -13,6 +13,7 @@ PLUGIN_LIST = \
 	mate        \
 	opcua       \
 	profinet    \
+	sap         \
 	stats_tree  \
 	unistim     \
 	wimax       \
