diff --git a/CMakeLists.txt b/CMakeLists.txt
index 245f322..9bb1b90 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1119,6 +1119,7 @@ if(ENABLE_PLUGINS)
 		plugins/mate
 		plugins/opcua
 		plugins/profinet
+		plugins/sap
 		plugins/stats_tree
 		plugins/unistim
 		plugins/wimax
diff --git a/Makefile.am b/Makefile.am
index cc40fe8..f734adc 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -354,6 +354,7 @@ plugin_ldadd = $(_CUSTOM_plugin_ldadd_) \
 	-dlopen plugins/mate/mate.la \
 	-dlopen plugins/opcua/opcua.la \
 	-dlopen plugins/profinet/profinet.la \
+	-dlopen plugins/sap/sap.la \
 	-dlopen plugins/stats_tree/stats_tree.la \
 	-dlopen plugins/unistim/unistim.la \
 	-dlopen plugins/wimax/wimax.la \
diff --git a/configure.ac b/configure.ac
index edead79..fa250ef 100644
--- a/configure.ac
+++ b/configure.ac
@@ -2827,6 +2827,7 @@ AC_CONFIG_FILES(
   plugins/mate/Makefile
   plugins/opcua/Makefile
   plugins/profinet/Makefile
+  plugins/sap/Makefile
   plugins/stats_tree/Makefile
   plugins/unistim/Makefile
   plugins/wimax/Makefile
diff --git a/epan/Makefile.am b/epan/Makefile.am
index 1d3a9ba..0cfe750 100644
--- a/epan/Makefile.am
+++ b/epan/Makefile.am
@@ -491,6 +491,13 @@ plugin_src = \
 	../plugins/irda/packet-irda.c \
 	../plugins/m2m/packet-m2m.c \
 	../plugins/m2m/wimax_tlv.c \
+	../plugins/sap/src/packet-sapdiag.c \
+	../plugins/sap/src/packet-sapenqueue.c \
+	../plugins/sap/src/packet-saprouter.c \
+	../plugins/sap/src/packet-sapprotocol.c \
+	../plugins/sap/src/packet-saprfc.c \
+	../plugins/sap/src/packet-sapms.c \
+	../plugins/sap/src/packet-sapsnc.c \
 	../plugins/wimax/crc.c \
 	../plugins/wimax/crc_data.c \
 	../plugins/wimax/mac_hd_generic_decoder.c \
diff --git a/packaging/nsis/wireshark.nsi b/packaging/nsis/wireshark.nsi
index 33d300d..67a1668 100644
--- a/packaging/nsis/wireshark.nsi
+++ b/packaging/nsis/wireshark.nsi
@@ -961,6 +961,7 @@ File "${STAGING_DIR}\plugins\irda.dll"
 File "${STAGING_DIR}\plugins\m2m.dll"
 File "${STAGING_DIR}\plugins\opcua.dll"
 File "${STAGING_DIR}\plugins\profinet.dll"
+File "${STAGING_DIR}\plugins\sap.dll"
 File "${STAGING_DIR}\plugins\unistim.dll"
 File "${STAGING_DIR}\plugins\wimax.dll"
 File "${STAGING_DIR}\plugins\wimaxasncp.dll"
diff --git a/plugins/Makefile.am b/plugins/Makefile.am
index 6b11b4e..fbbfb1e 100644
--- a/plugins/Makefile.am
+++ b/plugins/Makefile.am
@@ -31,6 +31,7 @@ SUBDIRS = $(_CUSTOM_SUBDIRS_) \
 	mate \
 	opcua \
 	profinet \
+	sap \
 	stats_tree \
 	unistim \
 	wimax \
